skimage/restoration/_cycle_spin.py

```diff
@@ -155,5 +155,5 @@ def _run_one_shift(shift):
         # multithreaded via dask
         futures = [dask.delayed(_run_one_shift)(s) for s in all_shifts]
         mean = sum(futures) / len(futures)
-        mean = mean.compute(get=dask.threaded.get, num_workers=num_workers)
+        mean = mean.compute(scheduler='threading', num_workers=num_workers)
```

Comment:
To be fixed via https://github.com/scikit-image/scikit-image/pull/3205

Reply:
Ok. That was easy to remove using rebasing :D